app/routes/_libraries/config.$version.index.tsx

```diff
@@ -30,159 +30,163 @@ const librariesRouteApi = getRouteApi('/_libraries')
 
 export default function FormVersionIndex() {
   const { sponsorsPromise } = librariesRouteApi.useLoaderData()
-  const { version } = Route.useParams()
   const library = getLibrary('config')
 
   const gradientText = `pr-1 inline-block leading-snug text-transparent bg-clip-text bg-gradient-to-r ${configProject.colorFrom} ${configProject.colorTo}`
 
   return (
-    <>
-      <div className="flex flex-col gap-20 md:gap-32 pt-32">
-        <div className="flex flex-col items-center gap-8 text-center px-4">
-          <h1 className="font-black flex gap-3 items-center text-4xl md:text-6xl lg:text-7xl xl:text-8xl uppercase [letter-spacing:-.05em]">
-            <span>TanStack</span>
-            <span className={twMerge(gradientText)}>Config</span>
-          </h1>
-          <h2
-            className="font-bold text-2xl max-w-[600px]
-            md:text-3xl
-            lg:text-5xl lg:max-w-[800px]"
-          >
-            <span className="underline decoration-dashed decoration-gray-500 decoration-3 underline-offset-2">
-              Configuration and tools
-            </span>{' '}
-            for publishing and maintaining high-quality JavaScript packages
-          </h2>
-          <Link
-            to="./docs/"
-            className={`py-2 px-4 bg-gray-500 text-white rounded uppercase font-extrabold`}
-          >
-            Get Started
-          </Link>
-        </div>
+    <div className="flex flex-1 flex-col min-h-0 relative overflow-x-hidden">
+      <div className="flex flex-1 min-h-0 relative justify-center overflow-x-hidden">
+        <div className="flex flex-col gap-20 md:gap-32 max-w-full py-32">
```

Comment:
Pretty much all the styles are a copy and paste job from `_libraries/query.$version.index.tsx`. I don't know if I like all the div nesting but it works there and we're not consistent at least.